code/src/Components/Header.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { API_KEY } from '../urls';
+
+const optionsArray = [
+    { option: 'Popular', value: 'popular' },
+    { option: 'Top Rated', value: 'top_rated' },
+    { option: 'Now Playing', value: 'now_playing' },
+    { option: 'Upcoming', value: 'upcoming'}
+];
+
+const Header = ({ reachAPIValue }) => {
+
+    const onOptionChange = (value) => {
```

Comment:
Really nice code and I love that you've implemented more features  in your application, good job 👏